src/main/resources/application.conf

```diff
@@ -43,6 +43,9 @@ backend {
   //  baseExecutionBucket = ""
   //  endpointUrl = ""
   //}
+  local {
+    tryToHardlinkInputs = true
+  }
```

Comment:
Ah right - I was thinking the `cromwell-executions` was hardwired and glommed on to the base execution bucket in the JES conf. 

Either way I think this is all out of scope here